sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/hash.scala

```diff
@@ -647,7 +652,8 @@ object XxHash64Function extends InterpretedHashFunction {
  * we can guarantee shuffle and bucketing have same data distribution
  */
 @ExpressionDescription(
-  usage = "_FUNC_(expr1, expr2, ...) - Returns a hash value of the arguments.")
+  usage = "_FUNC_(expr1, expr2, ...) - Returns a hash value of the arguments.",
+  since = "2.2.0")
 case class HiveHash(children: Seq[Expression]) extends HashExpression[Int] {
```

Comment:
This is not exposed in the `FunctionRegistry`. Is that intentional?

Reply:
Yea, I think yes.